src/tab/scss/variable.scss

```diff
@@ -183,10 +183,14 @@ $tab-pure-divider-line: $tab-pure-divider-border-style $tab-pure-divider-line-wi
 /// @namespace size/tab
 $tab-wrapped-border-line-style: $line-solid !default;
 
-/// broder width
+/// active border width
 /// @namespace size/tab
 $tab-wrapped-border-line-width: $line-1 !default;
 
+/// non-active border width
+/// @namespace size/tab
+$tab-wrapped-border-side-width: $line-1 !default;
+
```

Comment:
```
/// border width (active)
/// @namespace size/tab
$tab-wrapped-border-line-width: $line-1 !default;

/// border width (non-active)
/// @namespace size/tab
$tab-wrapped-border-side-width: $line-1 !default;
```

I suggest using `border width (active)` `border width (non-active)` so that they are next to each other

Reply:
Accepted

Reply:
Changed